de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5556570_sys_gh6458_Fix_C_InvoiceLines_With_Invalid_C_Uom_ID_or_Price_Uom_ID.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE c_invoiceline il
```

Comment:
pls add UpdatedBy and Updated with a "hardcoded" timestamp, e.g. `2020-04-09 13:23:07.035603+02`
on some DBs there might be "a lot" of C_InvoiceLines affected by this SQL

Reply:
done